no python binding generators found

I'm continuing on rebuilding ros-melodic on arch now that this effort is going on. I had some lingering previous packages like `pyside2`, `python-pyside2`, `shiboken2`, and `python-shiboken2` that came up during my previous adventure. I was getting this error ([mentioned on AUR](https://aur.archlinux.org/packages/ros-melodic-qt-gui-cpp/) as well) when trying to build this package:

```
/bin/sh: --generatorSet=shiboken: command not found
```
In looking for that:
```
$ grep -r "\-\-generatorSet" *
src/build/src/qt_gui_cpp_shiboken/CMakeFiles/qt_gui_cpp_shiboken.dir/build.make:	cd /home/jwhendy/installed/build/ros-melodic-aur/ros-melodic-qt-gui-cpp/src/qt_gui_core-0.3.11/qt_gui_cpp/src/qt_gui_cpp_shiboken && --generatorSet=shiboken --include-paths=/home/jwhendy/installed/build/ros-melodic-aur/ros-melodic-qt-gui-cpp/src/qt_gui_core-0.3.11/qt_gui_cpp/src/qt_gui_cpp_shiboken/../../include:/usr/include/qt/:/usr/include/qt/QtWidgets:/usr/include/qt/QtGui:/usr/include/qt/QtCore:/usr/lib/qt//mkspecs/linux-g++:/usr/include --typesystem-paths=/usr/share/PySide2/typesystems --output-directory=/home/jwhendy/installed/build/ros-melodic-aur/ros-melodic-qt-gui-cpp/src/build/src/qt_gui_cpp_shiboken global.h typesystem.xml
```

I think that command is likely supposed to be:
```
... && /path/to/bin/shiboken --generatorSet=shiboken ...
```

This [older post](https://github.com/ros-visualization/qt_gui_core/issues/35#issuecomment-28527836) suggested removing `shiboken`, so I did that.

Now I'm getting this:
```
CMake Warning at /opt/ros/melodic/share/python_qt_binding/cmake/shiboken_helper.cmake:18 (find_package):
  By not providing "FindShiboken2.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Shiboken2", but CMake did not find one.

  Could not find a package configuration file provided by "Shiboken2" with
  any of the following names:

    Shiboken2Config.cmake
    shiboken2-config.cmake

  Add the installation prefix of "Shiboken2" to CMAKE_PREFIX_PATH or set
  "Shiboken2_DIR" to a directory containing one of the above files.  If
  "Shiboken2" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  src/qt_gui_cpp_shiboken/CMakeLists.txt:41 (include)


CMake Warning at /opt/ros/melodic/share/python_qt_binding/cmake/shiboken_helper.cmake:23 (find_package):
  By not providing "FindPySide2.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "PySide2", but
  CMake did not find one.

  Could not find a package configuration file provided by "PySide2" with any
  of the following names:

    PySide2Config.cmake
    pyside2-config.cmake

  Add the installation prefix of "PySide2" to CMAKE_PREFIX_PATH or set
  "PySide2_DIR" to a directory containing one of the above files.  If
  "PySide2" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  src/qt_gui_cpp_shiboken/CMakeLists.txt:41 (include)

CMake Warning at /opt/ros/melodic/share/python_qt_binding/cmake/shiboken_helper.cmake:34 (message):
  Shiboken binding generator NOT available.
Call Stack (most recent call first):
  src/qt_gui_cpp_shiboken/CMakeLists.txt:41 (include)

-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.7.3", minimum required is "3.7") 
CMake Warning at /opt/ros/melodic/share/python_qt_binding/cmake/sip_helper.cmake:18 (message):
  SIP binding generator NOT available.
Call Stack (most recent call first):
  src/qt_gui_cpp_sip/CMakeLists.txt:36 (include)

-- Python binding generators: 
CMake Error at src/CMakeLists.txt:10 (message):
  No Python binding generator found.
```

Then I installed `sip`, and now I get:
```
Traceback (most recent call last):
  File "/opt/ros/melodic/share/python_qt_binding/cmake/sip_configure.py", line 8, in <module>
    import sipconfig
ModuleNotFoundError: No module named 'sipconfig'
make[2]: *** [src/qt_gui_cpp_sip/CMakeFiles/libqt_gui_cpp_sip.dir/build.make:90: sip/qt_gui_cpp_sip/Makefile] Error 1
make[1]: *** [CMakeFiles/Makefile2:380: src/qt_gui_cpp_sip/CMakeFiles/libqt_gui_cpp_sip.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

I'm pretty confused by what to make of this. If pyside and/or shiboken and/or sip are required, they should be [make]depends, right? On top of that, something else doesn't seem to be acting right.

--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = ros-melodic-qt-gui-cpp
 	pkgdesc = ROS - qt_gui_cpp provides the foundation for C++-bindings for qt_gui and creates bindings for every generator available.
 	pkgver = 0.3.11
-	pkgrel = 3
+	pkgrel = 4
 	url = https://wiki.ros.org/qt_gui_cpp
 	arch = any
 	license = BSD
@@ -17,6 +17,8 @@ pkgbase = ros-melodic-qt-gui-cpp
 	depends = ros-melodic-qt-gui
 	depends = ros-melodic-pluginlib
 	depends = tinyxml
+	depends = sip
+	depends = python-sip
 	source = ros-melodic-qt-gui-cpp-0.3.11.tar.gz::https://github.com/ros-visualization/qt_gui_core/archive/0.3.11.tar.gz
 	sha256sums = 708873d6b5a2e4543435e8a1a2349de8eb41937aaf78b9dbcbe08b4a1dc12da2
 

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -6,7 +6,7 @@ url='https://wiki.ros.org/qt_gui_cpp'
 pkgname='ros-melodic-qt-gui-cpp'
 pkgver='0.3.11'
 arch=('any')
-pkgrel=3
+pkgrel=4
 license=('BSD')
 
 ros_makedepends=(
@@ -33,6 +33,8 @@ ros_depends=(
 depends=(
 	${ros_depends[@]}
 	tinyxml
+    sip
+    python-sip
 )
 
 _dir="qt_gui_core-${pkgver}/qt_gui_cpp"

